Core.ApplicationServices/Contract/ReadModels/GridLocalItContractRolesService.cs

```diff
@@ -2,65 +2,24 @@
 using System.Collections.Generic;
 using System.Linq;
 using Core.Abstractions.Types;
+using Core.ApplicationServices.OptionTypes;
 using Core.DomainModel.ItContract;
-using Core.DomainModel.LocalOptions;
-using Core.DomainModel.Organization;
-using Core.DomainServices.Generic;
-using Core.DomainServices;
-using Core.DomainServices.Extensions;
 
 namespace Core.ApplicationServices.Contract.ReadModels
 {
     public class GridLocalItContractRolesService : IGridLocalItContractRolesService
     {
-        private readonly IEntityIdentityResolver _entityIdentityResolver;
-        private readonly IGenericRepository<LocalItContractRole> _repository;
-        private readonly IGenericRepository<ItContractRole> _optionsRepository;
+        private readonly IOptionsApplicationService<ItContractRight, ItContractRole> _optionService;
 
-        public GridLocalItContractRolesService(IEntityIdentityResolver entityIdentityResolver, IGenericRepository<LocalItContractRole> repository, IGenericRepository<ItContractRole> optionsRepository)
+        public GridLocalItContractRolesService(IOptionsApplicationService<ItContractRight, ItContractRole> optionService)
         {
-            _entityIdentityResolver = entityIdentityResolver;
-            _repository = repository;
-            _optionsRepository = optionsRepository;
+            _optionService = optionService;
         }
 
         public Result<IEnumerable<ItContractRole>, OperationError> GetOverviewRoles(Guid organizationUuid)
```

Comment:
Can we just ditch this whole service and use the IOptionsApplicationService in the Controller?